keras/layers/convolutional.py

```diff
@@ -3400,6 +3400,12 @@ def compute_output_shape(self, input_shape):
     return tf.TensorShape([input_shape[0], length, input_shape[2]])
 
   def call(self, inputs):
+    if sum(self.cropping) >= inputs.shape[1]:
+      raise ValueError(
```

Comment:
The ValueError needs to be tested with a test that uses `assertRaisesRegex`

Reply:
Sorry, just to confirm. Do you mean about create a new tests for this `ValueError`?